nixos/modules/rename.nix

```diff
@@ -72,6 +72,7 @@ with lib;
     (mkRemovedOptionModule [ "security" "setuidPrograms" ] "Use security.wrappers instead")
 
     (mkRenamedOptionModule [ "security" "virtualization" "flushL1DataCache" ] [ "security" "virtualisation" "flushL1DataCache" ])
+    (mkRenamedOptionModule ["security" "initialRootPassword"] ["users" "users" "root" "initialHashedPassword"])
```

Comment:
please add this to `nixos/modules/config/users-groups.nix` instead, like done in other places - we intend to move away from a global `rename.nix`.